### PR TITLE
Updating sr readme, adding raws, & batch info

### PR DESCRIPTION
**Pull request by jarumihooi**

### CODE REVIEW

**Comment by jarumihooi** [comment]
Added raws to correct batches. double checked batches. 
removed `cpb-aacip-516-5m6251gg3j` from one of the batches, this is the 42nd file that was not delivered in October.

**Review by keighrim — APPROVED** [review]
I added some clarification, attempting to put distinction between SR as an annotation project and SWT as an app derived from the data. Feel free to edit more if you feel so, but I think overall it's ready to merge.

File: scene-recognition/readme.md [hunk 1]
```diff
@@ -75,8 +75,8 @@ The subtypes of slates (blue) is also important to annotate.
 However, the non-important cases (grey) are various different negative cases that are not frames of interest. These may be similar to positive cases. These are sometimes less distinct between each other. Do the best possible, but move on if too much time is spent figuring out the distinctions.  
 Add the [modifier](https://docs.google.com/document/d/1IyM_rCsCr_1XQ39j36WMX-XnVVBT4T_01j-M0eYqyDs/edit#heading=h.xnfilznsrhpe) where needed. I.e. Pick the most preferred, clearest `type label`, add "Shift" when making the key combo. 
 
-### How to Annotate It
-The tool uses one or two key-combination presses to annotate the different kinds of frames. A key combination can be a single key, or could be a combo like "Shift P". Press the relevant one to annotate the `type`. 
+### How to Annotate It (Annotators)
```
Comment (jarumihooi) [inline scene-recognition/readme.md:78]: Is there any reason for this line to be plural and line51 to be singular? 

There is a minor typo that doesn't hurt readability in line47"need"-> "needs".

Comment (keighrim) [reply scene-recognition/readme.md:78]: I guess this was also a typo.

Comment (jarumihooi) [reply scene-recognition/readme.md]: I've removed those additions to the section headers, the preparation step makes its clear the Pm's work is to do the preparation. The rest is annotator work as usual.

**Review by keighrim — APPROVED** [review]

**Comment by jarumihooi** [comment]
fixes #66

**Pull request merged**